mmv1/products/iamworkforcepool/WorkforcePoolProviderScimToken.yaml

```diff
@@ -0,0 +1,111 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'WorkforcePoolProviderScimToken'
+description: |
+  Represents a SCIM Token for a Workforce Pool Provider Scim Tenant.
+  The SCIM Token is used for authenticating SCIM provisioning requests during the synchronization of user/group identities from external identity provider into Google Cloud using the System for Cross-domain Identity Management (SCIM) protocol. This needs to be provided in the Secret (Long Lived) Token field when configuring SCIM on an IdP.
+references:
+  guides:
+    'QUICKSTART_TITLE': 'https://cloud.google.com/iam/docs/workforce-sign-in-microsoft-entra-id-scalable-groups?group_type=extended#extended-attributes'
+  api: 'https://cloud.google.com/sdk/gcloud/reference/iam/workforce-pools/providers/scim-tenants/tokens'
+base_url: 'locations/{{location}}/workforcePools/{{workforce_pool_id}}/providers/{{provider_id}}/scimTenants/{{scim_tenant_id}}/tokens'
+self_link: 'locations/{{location}}/workforcePools/{{workforce_pool_id}}/providers/{{provider_id}}/scimTenants/{{scim_tenant_id}}/tokens/{{scim_token_id}}'
+create_url: 'locations/{{location}}/workforcePools/{{workforce_pool_id}}/providers/{{provider_id}}/scimTenants/{{scim_tenant_id}}/tokens?workforcePoolProviderScimTokenId={{scim_token_id}}'
+
+import_format:
+  - 'locations/{{location}}/workforcePools/{{workforce_pool_id}}/providers/{{provider_id}}/scimTenants/{{scim_tenant_id}}/tokens/{{scim_token_id}}'
+update_verb: 'PATCH'
+update_mask: true
+timeouts:
+  insert_minutes: 20
+  delete_minutes: 20
+custom_code:
+  decoder: 'templates/terraform/decoders/treat_deleted_state_as_gone.go.tmpl'
+  post_create: 'templates/terraform/post_create/sleep.go.tmpl'
+  post_update: 'templates/terraform/post_create/sleep.go.tmpl'
+examples:
+  - name: 'iam_workforce_pool_provider_scim_token_basic'
+    primary_resource_id: "example"
+    vars:
+      workforce_pool_id: 'example-pool'
+      provider_id: 'example-prvdr'
+      scim_tenant_id: 'example-tenant'
+      scim_token_id: 'example-scim-token'
+    test_env_vars:
+      org_id: 'ORG_ID'
+properties:
+  - name: 'name'
+    type: String
+    description: |
+      Identifier. The resource name of the scim token.
+      Format: `locations/{location}/workforcePools/{workforce_pool}/providers/{workforce_pool_provider}/scimTenants/{scim_tenant_id}/tokens/{scim_token_id}`
+    output: true
+  - name: 'displayName'
+    type: String
+    description:
+      A user-specified display name for the scim token. Cannot exceed 32 characters.
+  - name: 'securityToken'
+    type: String
+    description:
+      The token string provided to the IdP for authentication and will be set only during creation.
+    output: true
+  - name: 'state'
+    type: Enum
+    description: |
+      The current state of the scim token.
+      * STATE_UNSPECIFIED: State unspecified.
```

Comment:
generally we don't document UNSPECIFIED values because they're not meaningful to users.


```suggestion
```